src/main/java/school/lemon/changerequest/java/introduction/hw2/Task3.java

```diff
@@ -2,7 +2,17 @@
 
 
 public class Task3 {
-    public static String comboString(String s1, String s2) {
-        return "";
+    public static String comboString(String s1, String s2)
+    {
+        if(s1 == null)
+            return s1+s2+s1;
+        else
+        if(s2 == null)
+            return s2 + s1 + s2;
+        else
+        if(s1.length() < s2.length())
+            return s1+s2+s1;
+
+                return s2+s1+s2;
```

Comment:
It's OK, but format your code before submitting.
In IDEA you can use `CTRL+ALT+L` shortcut.